### PR TITLE
Dev/component

### PR DESCRIPTION
**Pull request by feriljohn**
add shapes to the canvas area

### CODE REVIEW

File: diagramming-tool/package-lock.json
Comment (AnushiyaM11) [inline diagramming-tool/package-lock.json:1]: Please ignore the changes in package-lock.json while committing the changes to the branch from next commit

File: .vscode/settings.json
Comment (AnushiyaM11) [inline .vscode/settings.json:1]: You can ignore this file in the next commit

File: diagramming-tool/.gitignore
Comment (AnushiyaM11) [inline diagramming-tool/.gitignore:1]: Remove the redundant lines of code

File: diagramming-tool/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <Router>
+      <div>
+        <header className="App-header">
+         
+        </header>
+        <Routes>
+          <Route exact path="/" element={<Home/>} />
+          <Route path="/login" element={<Login />} />
+          <Route path="/register" element={<Registration />} />
+          <Route path="/components" element={<HomeComponent  />} />
```
Comment (AnushiyaM11) [inline diagramming-tool/src/App.jsx:19]: Change the route path to some meaningful name

File: diagramming-tool/src/Component/HomePage/HomePage.js
Comment (AnushiyaM11) [inline diagramming-tool/src/Component/HomePage/HomePage.js:1]: HomePage.jsx

File: diagramming-tool/src/Component/Register/image.jpg
Comment (AnushiyaM11) [inline diagramming-tool/src/Component/Register/image.jpg:1]: Please move the image files to assets folder

**Comment by feriljohn** [comment]
created new branch

**Pull request closed without merging**